### PR TITLE
Fix error "spawn npm ENOENT" under Windows

### PR DESCRIPTION
**Pull request by forresst**
Following a discussion in [gitter](https://gitter.im/avajs/ava?at=577cdefc35f871276ad32634), an error npm "Error: spawn npm ENOENT"

Command-line input:

```console
ava --init
```

Error Message & Stack Trace:

```console
Error: spawn npm ENOENT
    at exports._errnoException (util.js:1007:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:182:32)
    at onErrorNT (internal/child_process.js:348:16)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)

    at Module.runMain (module.js:577:11)
    at run (node.js:348:7)
    at startup (node.js:140:9)
    at node.js:463:3
```

TODO:
- [X] check works under linux and Mac
- [X] add several tests (Added AppVeyor by sindresorhus for test under Windows)

I tested this fix the problem under windows. But it's a quick fix, your opinion interests me.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Looks good. I've been meaning to switch to `cross-spawn` too.

**Comment by forresst** [comment]
@sindresorhus To improve testing, I think the right solution is to add the Appveyor or something similar. What is the procedure? (I can handle the appveyor.yml file but made the necessary in Appveyor)

File: index.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ module.exports = function (opts) {
 		return Promise.resolve();
 	}
 
-	var child = childProcess.spawn('npm', ['install', '--save-dev', 'ava'], {
+	var child = spawn('npm', ['install', '--save-dev', 'ava'], {
```
Comment (sindresorhus) [inline index.js]: Can you add a `TODO` comment here about using https://github.com/sindresorhus/execa when we eventually target Node.js >=4

**Comment by sindresorhus** [comment]
I've added AppVeyor. Just rebase from master.

File: index.js (hunk 1 above)
Comment (forresst) [inline index.js]: Yep. I'll do.

**Comment by forresst** [comment]
@sindresorhus Done

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**